api/serializers/user.js

```diff
@@ -1,23 +1,33 @@
-const { Build, User, Site } = require('../models');
+const { User, UserAction, ActionType } = require('../models');
 
-const serializeObject = (user) => {
-  const json = user.toJSON();
-  json.sites = user.Sites.map(site => site.toJSON());
-  json.builds = user.Builds.map(build => build.toJSON());
-  delete json.Sites;
-  delete json.Builds;
-  return json;
-};
+const serializeObject = user => user.toJSON();
 
 const serialize = (serializable) => {
   if (serializable.length !== undefined) {
     const userIds = serializable.map(user => user.id);
-    const query = User.findAll({ where: { id: userIds }, include: [Site, Build] });
+    const query = User.findAll({ where: { id: userIds } });
```

Comment:
I removed the eager loading of `Site` and `Build` because we don't use them at all on the frontend